config/rewrites.d/api.rackspace.com.json

```diff
@@ -33,105 +33,105 @@
           "rewrite": false,
           "status": 301
       },
       {
           "description": "Cloud DNS redirect",
           "from": "^\\/api-ref-dns(\\.html)?",
-          "to": "/docs/cloud-dns/v1/developer-guide/",
+          "to": "/docs/cloud-dns/v1/api-reference/",
           "rewrite": false,
           "status": 301
       },
       {
           "description": "Cloud Files redirect",
           "from": "^\\/api-ref-files(\\.html)?",
-          "to": "/docs/cloud-files/v1/developer-guide/",
+          "to": "/docs/cloud-files/v1/api-reference/",
           "rewrite": false,
           "status": 301
       },
       {
           "description": "Cloud Images redirect",
           "from": "^\\/api-ref-images(\\.html)?",
-          "to": "/docs/cloud-images/v2/developer-guide/",
+          "to": "/docs/cloud-images/v2/api-reference/",
           "rewrite": false,
           "status": 301
       },
       {
           "description": "Cloud Load Balancers redirect",
           "from": "^\\/api-ref-load-balancers(\\.html)?",
-          "to": "/docs/cloud-load-balancers/v1/developer-guide/",
+          "to": "/docs/cloud-load-balancers/v1/api-reference/",
           "rewrite": false,
           "status": 301
       },
       {
           "description": "Cloud Monitoring redirect",
           "from": "^\\/api-ref-monitoring(\\.html)?",
-          "to": "/docs/cloud-monitoring/v1/developer-guide/",
+          "to": "/docs/rackspace-monitoring/v1/api-reference/",
           "rewrite": false,
           "status": 301
       },
       {
           "description": "Cloud Networks redirect",
           "from": "^\\/api-ref-networks(\\.html)?",
-          "to": "/docs/cloud-networks/v2/developer-guide/",
+          "to": "/docs/cloud-networks/v2/api-reference/",
           "rewrite": false,
           "status": 301
       },
       {
           "description": "Cloud Orchestration redirect",
           "from": "^\\/api-ref-orchestration(\\.html)?",
-          "to": "/docs/cloud-orchestration/v1/developer-guide/",
+          "to": "/docs/cloud-orchestration/v1/api-reference/",
           "rewrite": false,
           "status": 301
       },
       {
           "description": "Cloud Queues redirect",
           "from": "^\\/api-ref-queues(\\.html)?",
-          "to": "/docs/cloud-queues/v1/developer-guide/",
+          "to": "/docs/cloud-queues/v1/api-reference/",
           "rewrite": false,
           "status": 301
       },
       {
           "description": "Rackspace CDN redirect",
           "from": "^\\/api-ref-raxCDN(\\.html)?",
-          "to": "/docs/cdn/v1/developer-guide/",
+          "to": "/docs/cdn/v1/api-reference/",
           "rewrite": false,
           "status": 301
       },
       {
           "description": "Email and Apps API redirect",
           "from": "^\\/api-ref-email-and-apps(\\.html)?",
-          "to": "/docs/",
+          "to": "http://api-wiki.apps.rackspace.com/api-wiki/index.php/Main_Page",
```

Comment:
We never completely migrated the e-mail and apps API reference to docbook, so it was not migrated to Sphinx.  The REST API is not actively maintained, so pointing to the most current information that we have on the wiki.  I got the wiki link from the community website.